ios/Framework/Core/Context/Storage/OAuth2CredentialsStoreKeyChain.swift

```diff
@@ -0,0 +1,78 @@
+/**
+* Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+*
+* This library is free software; you can redistribute it and/or modify it under
+* the terms of the GNU Lesser General Public License as published by the Free
+* Software Foundation; either version 2.1 of the License, or (at your option)
+* any later version.
+*
+* This library is distributed in the hope that it will be useful, but WITHOUT
+* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+* FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+* details.
+*/
+import KeychainAccess
+
+open class OAuth2CredentialsStoreKeyChain: BaseCredentialsStoreKeyChain {
```

Comment:
I'd like to consider this customer feedback https://web.liferay.com/group/customer/support/-/support/ticket/MCDSCREENS-56